build.gradle

```diff
@@ -27,7 +27,7 @@ plugins {
     id 'com.github.johnrengelman.shadow' version '1.2.3'
     id 'com.palantir.configuration-resolver' version '0.1.0'
     id 'com.palantir.git-version' version '0.5.2'
-    id 'org.inferred.processors' version '1.1'
+    id 'org.inferred.processors' version '1.2.4-rc2'
```

Comment:
We should probably cut a real `1.2.4` with the changes https://github.com/palantir/gradle-processors/compare/v1.2.3...master

@chrisalice for SA